### PR TITLE
Upgrade to v71

### PR DESCRIPTION
**Pull request by chiaramooney**
### Why

Upgrade app to v71. Upgrade navigation version to v6. Downgrade reanimated package to 1.0 following blank app screen bug.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -60,7 +60,10 @@ const styles = StyleSheet.create({
 });
 
 function RenderDrawerItem(props, i: number) {
-  const isDrawerOpen = getIsDrawerOpenFromState(props.navigation.getState());
+  const isDrawerOpen =
+    getDrawerStatusFromState(props.navigation.getState()) == 'open'
+      ? true
+      : false;
```
Comment (chrisglein) [inline src/App.tsx:66]:
```suggestion
    getDrawerStatusFromState(props.navigation.getState()) == 'open';
```
Isn't the `? true : false` here redundant?

File: src/App.tsx [hunk 2]
```diff
@@ -88,8 +91,10 @@ function RenderDrawer(props) {
 }
 
 function CustomDrawerContent(props) {
-  const isDrawerOpen = getIsDrawerOpenFromState(props.navigation.getState());
-
+  const isDrawerOpen =
+    getDrawerStatusFromState(props.navigation.getState()) == 'open'
+      ? true
+      : false;
```
Comment (chrisglein) [inline src/App.tsx:97]:
```suggestion
    getDrawerStatusFromState(props.navigation.getState()) == 'open';
```

**Review by TatianaKapos — APPROVED** [review]
LGTM! :)

File: windows/rngallery/rngallery.vcxproj [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- This project was created with react-native-windows 0.68.0 -->
+<!-- This project was created with react-native-windows 0.71.0-preview.3 -->
```
Comment (chrisglein) [inline windows/rngallery/rngallery.vcxproj:2]: Why preview release?

File: __tests__/App-test.js [hunk 4]
```diff
@@ -20,7 +20,7 @@ import {TimePickerExamplePage} from '../src/examples/TimePickerExamplePage';
 // Issue: Usage of UIManager.getViewManangerConfig returning undefined
 //import {SketchExamplePage} from '../src/examples/SketchExamplePage';
 import {SliderExamplePage} from '../src/examples/SliderExamplePage';
-import {PermissionsExamplePage} from '../src/examples/PermissionsExamplePage';
+//import {PermissionsExamplePage} from '../src/examples/PermissionsExamplePage';
```
Comment (chrisglein) [inline __tests__/App-test.js:23]: Tracking issue #?

**Pull request merged**